shim/src/realm.js

```diff
@@ -165,15 +190,6 @@ export default class Realm {
     };
     Realm2RealmRec.set(O, realmRec);
 
-    const init = O.init;
-    if (!IsCallable(init)) throw new TypeError();
-    init.call(O);
```

Comment:
I still don't fully understand how are we removing `init`, and how do the user have control of the globals added to global during initialization.

It seems that the shim is moving faster than the spec, which is fine, but maybe it should not be part of this repo then. My main concern is that folks who come here to look at the spec, and try out the polyfill, will not realize that the polyfill is not following the spec.